api/config/cdr_config_perf.json

```diff
@@ -39,16 +47,34 @@
     {
       "cdrVersionId": 3,
       "isDefault": true,
+      "accessTier": "registered",
       "name": "Synthetic Dataset v3",
-      "bigqueryProject": "all-of-us-ehr-dev",
+      "bigqueryProject": "fc-aou-cdr-perf",
       "bigqueryDataset": "test_R2019q4r3",
-      "creationTime": "2019-09-30 00:00:00Z",
+      "creationTime": "2021-04-20 00:00:00Z",
       "releaseNumber": 3,
       "numParticipants": 234525,
       "cdrDbName": "synth_r_2019q4_14",
+      "elasticIndexBaseName": "cdr",
+      "microarrayBigqueryDataset": "microarray_data",
       "hasFitbitData": true,
-      "hasCopeSurveyData": true,
-      "accessTier": "registered"
+      "hasCopeSurveyData": true
+    },
+    {
+      "cdrVersionId": 4,
+      "isDefault": true,
+      "accessTier": "controlled",
+      "name": "CDR for Controlled Tier - DO NOT USE",
+      "bigqueryProject": "fc-aou-cdr-perf-ct",
+      "bigqueryDataset": "test_R2019q4r3",
```

Comment:
Not sure whether you created this dataset name or not, but this prefix looks abnormal to me: where did "test_" come from? Also usually the convention has been "SR2019q4r3", where:
- S = synthetic
- R = registered

Reply:
nvm, test env uses the same dataset names - unrelated to this PR then

Reply:
@calbach For the synth dataset incident, Kerri's team named the new dataset with test_ because they didn't want us to refer to it as synthetic. I'm assuming when we get our next version of the synthetic dataset, it will use the SR prefix.